ztp/kube-compare-reference/Makefile

```diff
@@ -24,4 +27,25 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1 /usr/local/bin/helm
+
+
+.PHONY: convert
+convert: $(HELMCONVERTOR) helm
+	@echo "Converting reference files to Helm Charts."
+	@$(HELMCONVERTOR) -r ./metadata.yaml -n Chartv1 -v default_value.yaml
+	@echo "Rendering Helm Charts to CR files."
+	@helm template renderedv1 ./Chartv1 --output-dir renderedv1
+
+$(HELMCONVERTOR):
+	@echo "Installing helm-convert tool..."
+	go install github.com/openshift/kube-compare/addon-tools/helm-convert@latest
+
+helm:
+	@echo "Installing helm..."
+	curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
```

Comment:
Can we ask this to install to an alternate location like `$pwd`?

Reply:
Yes, I changed the method to curl the binary directly instead of running installation script.